lib/rubocop/node_pattern.rb

```diff
@@ -440,11 +442,17 @@ def emit_trailing_params
         params.empty? ? '' : ",#{params}"
       end
 
+      def emit_guard_clause
+        <<-RUBY
+          return nil unless node.is_a?(RuboCop::AST::Node)
```

Comment:
or at least `return unless node.is_a?(RuboCop::AST::Node)`, as `return` and `return nil` are the same thing, right? 

BTW, is there a Rubocop cop for this?